weather_data_reader.py

```diff
@@ -0,0 +1,33 @@
+import calendar
+import os
+
+from parse_weather_data import WeatherParser
+from weather_calculation import WeatherCalculations
+
+
+class WeatherDataReader:
+
+    def __init__(self, directory):
+        self.directory = directory
+        self.parser = WeatherParser()
+        self.weather_stats = WeatherCalculations()
+        self.file_path = os.path.join(self.directory, "Murree_weather_{year}_{month}.txt")
+
+    def read_data(self, file_path):
```

Comment:
Which data are we talking about? Data is quite vague word when it comes to naming, be precise.